ebu_tt_live/scripts/ebu_ebuttd_encoder.py

```diff
@@ -34,13 +38,28 @@
                     help='Works only with -m, if set the script will wait for new lines to be added to the file once the last line is reached. Exactly like tail -f does.',
                     action="store_true", default=False
                     )
+parser.add_argument('--implicit-ns', help='Some tools hardcode tt so they can\'t understand tt:tt so global namespace disappears',
+                    action='store_true', dest='implicit_ns', default=False)
 parser.add_argument('-z', '--clock-at-media-time-zero', dest='media_time_zero',
                     help='This sets the offset value that is used to turn clock time into media time.',
                     default='current', metavar='HH:MM:SS.mmm')
+parser.add_argument('-zo', '--clock-at-media-time-zero-offset', dest='media_time_zero_offset',
+                    help='This sets the offset value that is used to turn clock time into media time.',
```

Comment:
ok the 2 flags ended up with the same text which is a but misleading I agree. -z is the one you talking about -zo is an adjustment to that. -zo is a tiny number usually -z is an explicit clock time